sample-clients/csharp/client/ScoringClient.cs

```diff
@@ -41,19 +42,20 @@ public ScoringClient(string host, int port, bool useSsl = false, string authKey
             _client = new PredictionServiceClientWrapper(new PredictionService.PredictionServiceClient(channel));
         }
 
-        public async Task<float[]> ScoreAsync(IScoringRequest request)
+        public async Task<float[]> ScoreAsync(IScoringRequest request, int retryCount = 20)
         {
-            return await ScoreAsync<float[]>(request);
+            return await ScoreAsync<float[]>(request, retryCount);
         }
 
-        public async Task<T> ScoreAsync<T>(IScoringRequest request) where T : class
+        public async Task<T> ScoreAsync<T>(IScoringRequest request, int retryCount = 20) where T : class
```

Comment:
Maybe it should be a const

Reply:
C# doesn't have const parameters? Are you suggesting that retry count shouldn't be an optional parameter?

Reply:
Nope. Just make a value to be a const
```c#
private const DefaultRetryCount  = 20
…
int retryCount = DefaultRetryCount 
```

Reply:
Oh - yeah it probably makes more sense as a member.